docs/streaming_endpoints.md

```diff
@@ -0,0 +1,81 @@
+# gRPC Streaming API (preview) {#ovms_docs_streaming_endpoints}
+
+## Introduction
+OpenVINO Model Server implements gRPC KServe extension which adds separate RPC for [bidirectional streaming](https://grpc.io/docs/what-is-grpc/core-concepts/#bidirectional-streaming-rpc) use cases. It means that besides unary RPC, where client sends a single request and gets back a single response, client is able to initiate connection and send/receive messages in any number and order using `ModelStreamInfer` procedure:
+
+```
+service GRPCInferenceService
+{
+  ...
+  // Standard
+  rpc ModelInfer(ModelInferRequest) returns (ModelInferResponse) {}
+  // Extension
+  rpc ModelStreamInfer(stream ModelInferRequest) returns (stream ModelStreamInferResponse) {}
+  ...
+}
+```
+
+This becomes very useful for serving [MediaPipe Graphs](./mediapipe.md). In unary inference RPC, each request is using separate and independent MediaPipe graph instance.
+However, in streaming inference RPC MediaPipe graph is created only once per the client session, and is reused by subsequent requests to the same gRPC stream.
+
+- This avoids graph initialization overhead and increases its overall throughput
+- Allows to preserve state between subsequent incoming requests
+- Enables source calculators to stream multiple responses without feeding input packets
```

Comment:
Hmm, can we put it in simpler words maybe? That point might be quite mysterious for someone who's not very familiar with mediapipe.